src/components/hoc/withMeeting.jsx

```
@@ -32,7 +33,9 @@ export default function withMeeting(WrappedComponent) {
     const {meetingDestination} = props;
     const meeting = useMeetingDestination(meetingDestination);
 
-    return <WrappedComponent meeting={meeting} {...props} />;
+    return <WebexMeetingProvider>
+      <WrappedComponent meeting={meeting} {...props} />;
+    </WebexMeetingProvider>    
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)